views/pages/funding/programme-detail.njk

```diff
@@ -6,15 +6,9 @@
 {% set bodyClass = "has-full-bleed-header" %}
 
 {% block title %}{{ title }} | {% endblock %}
-{% set heroImage = createHeroImage({
-    small: 'hero/working-families-small.jpg',
-    medium: 'hero/working-families-medium.jpg',
-    large: 'hero/working-families-large.jpg',
-    default: 'hero/working-families-medium.jpg'
-}) %}
-{% set socialImage = heroImage %}
 
 {% block content %}
+    {% set socialImage = heroImage %}
```

Comment:
Does this work here? I _think_ this needs to be defined outside/before the `content` block.

Reply:
Good catch